app/db_requests.py

```diff
@@ -0,0 +1,81 @@
+from sqlalchemy import MetaData, create_engine, cast, Date
+from contextlib import contextmanager
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import func
+
+
+class DBRequests:
+    def __init__(self, URL: str, TABLE_NAME: str):
+        engine = create_engine(URL)
+        self.session = sessionmaker(bind=engine)
+        meta = MetaData()
+        meta.reflect(bind=engine)
+        self.table = meta.tables.get(TABLE_NAME)
+
+    @contextmanager
+    def session_scope(self):
+        session = self.session()
+        try:
+            yield session
+            session.commit()
+        finally:
+            session.close()
+
+    def get_avg_queries_time_per_day(self, date: str):
+        with self.session_scope() as s:
+            query_durations = (
+                s.query(
+                    self.table.c.query_id,
+                    ((func.max(self.table.c.time) - func.min(self.table.c.time))).label(
```

Comment:
What will happen for the query that started to execute on previous day and finished to execute on this day?

Reply:
The request for database is incorrect. agree